apps/sms/views/shared/style/composer.css

```diff
@@ -4,14 +4,22 @@
 
   height: auto; /* overrides building blocks */
   min-height: 4rem;
-
```

Comment:
please keep this line :) we try to stick to https://github.com/necolas/idiomatic-css#declaration-order for the new CSS code and new lines are part of it :)